xs/t/25_transformationmatrix.t

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/perl
```

Comment:
Can you port this to the C++ tests after merging?

Reply:
sure, I'll open another PR when I'm done